packages/shared/providers.yaml

```diff
@@ -1960,7 +1960,7 @@ guru:
         - knowledge-base
     auth_mode: BASIC
     proxy:
-        base_url: https://api.getguru.com/api/v1
```

Comment:
We have to be careful with this for users who are using this in the proxy or scripts. This would be a breaking change  that we need to announce etc.

Reply:
We could have a second provider, not ideal though
We could introduce a `patch` props: 
```yaml
guru:
  proxy:
      base_url: https://api.getguru.com/api/v1
  patch
    20241022:
      base_url: https://api.getguru.com/
```

or the opposite to optimize for new integrations

```yaml
guru:
  proxy:
      base_url: https://api.getguru.com
  patch
    20241022:
      base_url: https://api.getguru.com/api/v1
```

So in the code we could use patch depending on the `created_at`